package.json

```diff
@@ -10,8 +10,11 @@
     "Suhas Karanth <sudo.suhas@gmail.com>",
     "Iiro Jäppinen <iiro@jappinen.fi> (https://iiro.fi)"
   ],
-  "bin": "index.js",
-  "files": ["index.js", "src"],
+  "main": "./src/index.js",
+  "bin": {
+    "lint-staged": "./bin/lint-staged"
```

Comment:
I don't think this named form is necessary, since we only have the single bin, so maybe change back to `"bin": "./bin/lint-staged"`

Reply:
Sure thing, I've replaced it with the simplified form.